homeassistant/components/braviatv/config_flow.py

```diff
@@ -105,10 +116,12 @@ async def async_step_authorize(self, user_input=None):
             else:
                 user_input[CONF_HOST] = self.host
                 user_input[CONF_MAC] = self.mac
-                return self.async_create_entry(title=self.title, data=user_input)
-
+                return self.async_create_entry(
+                    title=cast(str, self.title), data=user_input
```

Comment:
Annotate in constructor.
```suggestion
                    title=self.title, data=user_input
```